unsafe/src/test/java/org/apache/spark/unsafe/types/UTF8StringSuite.java

```diff
@@ -25,21 +25,30 @@
 public class UTF8StringSuite {
 
   private void checkBasic(String str, int len) throws UnsupportedEncodingException {
-    Assert.assertEquals(UTF8String.fromString(str).length(), len);
-    Assert.assertEquals(UTF8String.fromBytes(str.getBytes("utf8")).length(), len);
+    UTF8String s1 = UTF8String.fromString(str);
+    UTF8String s2 = UTF8String.fromBytes(str.getBytes("utf8"));
+    Assert.assertEquals(s1.length(), len);
+    Assert.assertEquals(s2.length(), len);
 
-    Assert.assertEquals(UTF8String.fromString(str).toString(), str);
-    Assert.assertEquals(UTF8String.fromBytes(str.getBytes("utf8")).toString(), str);
-    Assert.assertEquals(UTF8String.fromBytes(str.getBytes("utf8")), UTF8String.fromString(str));
+    Assert.assertEquals(s1.toString(), str);
+    Assert.assertEquals(s2.toString(), str);
+    Assert.assertEquals(s1, s2);
 
-    Assert.assertEquals(UTF8String.fromString(str).hashCode(),
-      UTF8String.fromBytes(str.getBytes("utf8")).hashCode());
+    Assert.assertEquals(s1.hashCode(), s2.hashCode());
+
+    Assert.assertEquals(s1.compare(s2), 0);
+    Assert.assertEquals(s1.compareTo(s2), 0);
+
+    Assert.assertEquals(s1.contains(s2), true);
```

Comment:
I think you can use `assertTrue` here.  As long as you're changing this file, you might as well change it to statically import the `assert*` methods.